R/get_data.R

```diff
@@ -22,9 +24,6 @@
 #' @examples
 #' \dontrun{
 #'  get_layer("bc_bound_hres")
-#'
-#'  # As a "Spatial" (sp) object
-#'  get_layer("watercourses_15M")
 #' }
 get_layer <- function(layer, class = c("sf", "sp"), ask = TRUE, force = FALSE) {
```

Comment:
agree that it could go here and as long as the message comes across, maybe it is ok that it is at this function level?